a11y/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:15.0.1
+
+LABEL maintainer="daniel.walford@ons.gov.uk"
+
+# Explicitly declare npm user is root (docker runs as root anyway) allows global package installation for npm, needed for @axe-core/cli
+RUN npm --global config set user root
```

Comment:
I think this could probably be combined with the npm call at the bottom of the file to reduce layers as they are related.